parent/pom.xml

```diff
@@ -1220,7 +1220,7 @@
                 <version>${version.h2.engine}</version>
                 <exclusions>
                     <exclusion>
-                        <groupId>org.wso2.eclipse.osgi</groupId>
+                        <groupId>org.eclipse.platform</groupId>
```

Comment:
Why exclude "org.eclipse.platform", This version of h2 wouldn't have a dependency to *.platform. Check the transitive dependency and which one is the correct one to exclude.